package.json

```
@@ -8,7 +8,7 @@
   },
   "license": "MPL-2.0",
   "engines": {
-    "node": ">= 10.13 < 11 || >= 12.14 < 13"
+    "node": ">= 12.14 < 13 || >= 14.16 < 17"
```

@de-laz 
```suggestion
    "node": ">= 12.14 < 13 || >= 14.16 < 15"
```
17 seems a bit high if the latest stable is 14

Are there corresponding changes for the docker images?
https://github.com/jembi/openhim-docker/blob/core/5/Dockerfile#L1